### DISCUSSION TITLE
Nosey Parker Parser in Defect Dojo

### DISCUSSION THREAD

**Discussion by tpat13** [praetorian-inc/noseyparker#124] [Show and tell]
FYI - I've been working with @munntjlx and the PR for a Nosey Parker parser in Defect Dojo is under review: https://github.com/DefectDojo/django-DefectDojo/pull/9067

The purpose of this is to be able to turn secrets found from Nosey Parker into Findings to track in Defect Dojo (the open source vulnerability management tool).

**Comment by bradlarsen** [comment]
Cool! I will keep an eye on this; glad for the integrations!
